snake/snake_game_khr.py

```diff
@@ -0,0 +1,150 @@
+from time import sleep
+import os
+from random import randint
+
+
+class Board:
+    def __init__(self, width, height, top_border="<", bottom_border=">", vert_border="^"):
+        self.snake = Snake()
+        self.top_boarders = top_border
+        self.bottom_boarders = bottom_border
+        self.vert_boarders = vert_border
+        self.width = width
+        self.height = height
+        self.board = self.init_board()
+
+    def init_board(self):
+        new_board = []
+        for i in range(self.height):
+            if i in {0}:
+                row = [self.top_boarders] * self.width
+                new_board.append(row)
+            elif i in {self.height - 1}:
+                row = [self.bottom_boarders] * self.width
+                new_board.append(row)
+            else:
+                row = []
+                for j in range(self.width):
+                    if j in {0, self.width - 1}:
+                        row.append(self.vert_boarders)
+                    else:
+                        row.append(" ")
+                new_board.append(row)
+        return new_board
+
+    def show(self):
+        for row in self.board:
+            print(' '.join(row))
+
+
+class Snake:
+    def __init__(self, symbol="o", position=(1, 1)):
+        self.symbol = symbol
+        self.body = [position]
+
+    def eat(self, position: tuple):
+        self.body.append(position)
+
+    def move(self, position: tuple):
+        self.body.append(position)
+        self.body.pop(0)
+
+
+class Apple:
+    def __init__(self, symbol='$', position=(2, 2)):
+        self.position = [position]
+        self.symbol = symbol
+
+
+class Game:
+    def __init__(self):
+        self.symbol = "o"
+        self.width = 20
```

Comment:
you want to pass this as args I believe, so you can create Game classes with different size of the board eventually (or somebody else who will use your code).